presto-parser/src/main/java/com/facebook/presto/sql/tree/FrameBound.java

```diff
@@ -76,6 +78,11 @@ public Type getType()
         return value;
     }
 
+    public Optional<Expression> getOriginalValue()
```

Comment:
The value gets replaces by a Symbol and is not available during plan printing and is extracted during execution time. I had a chat with Eric and he said that it was done to keep the frames dynamic. I am not sure if there is another way to access the frame start and end values through the Symbol of the child node.